project/src/main/java/com/google/sps/store/SearchStore.java

```diff
@@ -59,27 +66,22 @@ private void addKeywordsInTextToIndex(String eventId, String text, float weight)
 
     for (Keyword keyword : keywords) {
       float keywordRank = 0;
```

Comment:
nit: For consistency, if these two ranking variables are referring to the same kind of ranking, either make them both primitive or both objects?

Reply:
Done.

Reply:
Done - yes I agree.